WebDriverAgentLib/Categories/XCUIElement+FBAccessibility.m

```diff
@@ -18,7 +18,7 @@ @implementation XCUIElement (FBAccessibility)
 
 - (BOOL)fb_isAccessibilityElement
 {
-  return ([self fb_snapshotWithAttributes:@[FB_XCAXAIsElementAttributeName]] ?: self.fb_lastSnapshot).fb_isAccessibilityElement;
+  return ([self fb_snapshotWithAttributes:@[FB_XCAXAIsElementAttributeName]] ?: self.lastSnapshot).fb_isAccessibilityElement;
```

Comment:
`self.lastSnapshot` can be nil if the element has not been resolved first or if none of the functions that set it have been called, I think we should add a fallback to `fb_uniqueSnapshot` here.

Reply:
lastSnapshot is updated automatically when `fb_snapshotWithAttributes` is called